lib/defaultConfig.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  drupal: {
+    path: "./.breakpoints.config.yml",
```

Comment:
Is it a good idea to define a default path for the Drupal breakpoints file? According to the [docs](https://www.drupal.org/docs/theming-drupal/working-with-breakpoints-in-drupal), the file name is `myThemeOrModule.breakpoints.yml`. So it’s hard to set a sensible default.